server/globals.ts

```diff
@@ -16,3 +16,48 @@ declare let IPTools: typeof import('./ip-tools').IPTools;
 declare let Sockets: typeof import('./sockets');
 // let TeamValidator: typeof import('../sim/team-validator');
 declare let TeamValidatorAsync: typeof import('./team-validator-async');
+
+interface Group {
+	symbol: string,
+	id: string,
+	name: string,
+
+	root?: boolean,
+	inherit?: string,
+	jurisdiction?: string,
+	roomonly?: boolean,
+	globalonly?: boolean,
+	console?: boolean,
+	lockdown?: boolean,
+	hotpatch?: boolean,
+	ignorelimits?: boolean,
+	promote?: string,
```

Comment:
```suggestion
	promote?: string | boolean,
```

Any permission with a target can take either a custom jurisdiction, or `true` meaning to fall back to the `jurisdiction` key.

Reply:
I think that's missing from the documentation in `config.js`. I'll add that if that's the case

Reply:
Yeah, it could very well be missing. I'm kind of bad at documentation sometimes.